pytype/tests/py3/CMakeLists.txt

```diff
@@ -9,6 +9,16 @@ py_test(
     pytype.tests.test_base
 )
 
+py_test(
+  NAME
+    test_annotated
```

Comment:
nit: Could you call this test_typing_annotated? pytype has a lot of functional tests, and it's hard to tell at first glance what "test_annotated" might be testing.

Reply:
Of course. Done.